index.bs

```diff
@@ -155,20 +155,24 @@ Note, the above algorithm:
     it parses `Accept-CH` and adds the results to the environment setting object's client hints set as well as the Accept-CH cache.
     </div>
 
-<dfn>Accept-CH state</dfn> (`http-equiv="accept-ch"`) {#accept-ch-state-algo}
+<dfn>Accept-CH state</dfn> (`name="accept-ch"`) {#accept-ch-state-algo}
 --------
 
-Note: This pragma *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=].
+Note: This metadata *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=].
 
 1. If the <{meta}> element is not a child of a <{head}> element, then return.
 2. If the <{meta}> element has no <{meta/content}> attribute, or if that attribute's value is the empty string, then return.
-3. Let |settingsObject| be the <{meta}> element's [=relevant settings object=].
-4. If the result of executing [$Is an environment settings object contextually secure?$] on |settingsObject| is `"Not Secure"`, then return.
-5. Let |browsingContext| be |settingsObject|'s  [=environment settings object/global object=]'s [=Window/browsing context=].
-6. If the [=top-level browsing context=] does not equal |browsingContext|, abort these steps.
-7. Let |acceptCHValue| be the <{meta}> element's <{meta/content}> attribute's value.
-8. Parse |acceptCHValue| according to the [=Accept-CH=] header parsing rules, as a [=field-name=].
-     [=set/Append=] each parsed [=client hints token=] to |settingsObject|'s [=environment settings object/client hints set=].
+3. If the <{meta}> element has any <a href="https://dom.spec.whatwg.org/#concept-tree-preceding">preceding</a> <a href="https://dom.spec.whatwg.org/#concept-tree-sibling">sibling</a> <{link}> or <{script}> elements (or if any <{link}> or <{script}> elements have begun to execute), then return.
+4. Let |settingsObject| be the <{meta}> element's [=relevant settings object=].
+5. If the result of executing [$Is an environment settings object contextually secure?$] on |settingsObject| is `"Not Secure"`, then return.
+6. Let |browsingContext| be |settingsObject|'s  [=environment settings object/global object=]'s [=Window/browsing context=].
+7. If the [=top-level browsing context=] does not equal |browsingContext|, abort these steps.
+8. Let |acceptCHValue| be the <{meta}> element's <{meta/content}> attribute's value.
+9. Parse |acceptCHValue| according to the [[PERMISSIONS-POLICY|`serialized permission policy`]]'s parsing rules
+     while limiting [[PERMISSIONS-POLICY|`feature-identifier`]]s to [=client hints token=]s and disallowing [[PERMISSIONS-POLICY|`allow-list-value`]]s
+     to be `*` (`*` can still be inherited from the default policy or HTTP header permissions policy). [=set/Append=] each parsed `feature-identifier`
```

Comment:
Can we pull this parenthetical out of this step, and toss it in a Note just below step 9? https://tabatkins.github.io/bikeshed/#notes-etc